HTAN.model.csv

```diff
@@ -58,6 +58,9 @@ Bulk RNA-seq Level 3,Bulk RNA-seq gene expression matrices,,"Component, Filename
 Bulk WES Level 2,Bulk Whole Exome Sequencing aligned files and QC,,"Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Alignment Workflow Type, Genomic Reference, Genomic Reference URL, Index File Name, Average Base Quality, Average Insert Size, Average Read Length, Contamination, Contamination Error, Mean Coverage, Adapter Content, Basic Statistics, Encoding, Overrepresented Sequences, Per Base N Content, Per Base Sequence Content, Per Base Sequence Quality, Per Sequence GC Content, Per Sequence Quality Score, Per Tile Sequence Quality, Percent GC Content, Sequence Duplication Levels, Sequence Length Distribution, QC Workflow Type, QC Workflow Version, QC Workflow Link, MSI Workflow Link, MSI Score, MSI Status, Pairs On Diff CHR, Total Reads, Total Uniquely Mapped, Total Unmapped reads, Proportion Reads Duplicated, Proportion Reads Mapped, Proportion Targets No Coverage, Proportion Base Mismatch, Short Reads, Proportion Coverage 10x, Proportion Coverage 30X,Is lowest level",,FALSE,Sequencing,Bulk WES Level 1,,
 Bulk WES Level 3,Bulk Whole Exome Sequencing called variants,,"Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID, Genomic Reference, Genomic Reference URL, Germline Variants Workflow URL, Germline Variants Workflow Type, Somatic Variants Workflow URL, Somatic Variants Workflow Type, Somatic Variants Sample Type, Structural Variant Workflow URL, Structural Variant Workflow Type",,FALSE,Sequencing,Bulk WES Level 2,,
+Microarray Level 1,Microarray Level 1 refers to the raw text table of probe level intensities,,"Component, Filename, File Format, HTAN Data File ID, HTAN Participant ID, HTAN Parent Biospecimen ID, Nucleic Acid Source, Microarray Platform ID, Microarray Molecule, Microarray Label, Microarray Value Definition, Microarray Protocol Auxiliary File",,FALSE,Assay,Biospecimen,,
+Microarray Level 2,Microarray Level 2 provides a normalized matrix of values.,,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Parent Data File ID, HTAN Data File ID, Microarray Platform ID, Normalization Method",,FALSE,Assay,Microarray Level 1,,
+Microarray Protocol Auxiliary File,"Auxiliary file describing the experimental protocols used, as described in the NCBI GEO microarray template, recorded as synapse ID (syn12345).",,"Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID",,FALSE,Assay,,,

```

Comment:
@Gibbsdavidl is this implemented as intended? Here this creates a new component called `Microarray Protocol Auxiliary File` but the description and RFC suggest it should be implemented as a attribute within `Microarray Level 1` that would take a synapse ID.

If the latter is the intention I suggest removing the entries in the `DependsOn` column and adding `regex match syn\d+` as the validation rule.

The RFC also suggests this is required